web/models/workspace.ex

```diff
@@ -32,15 +31,8 @@ defmodule PhoenixToggl.Workspace do
     |> foreign_key_constraint(:user_id)
   end
 
```

Comment:
I could not find a way to get this working without the hooks (removed in ecto v2). [prepare_changes](https://hexdocs.pm/ecto/Ecto.Changeset.html#prepare_changes/2) would not work here, since the `workspace.id` is not present at that point.